src/UnitySampleProject/Assets/Scripts/Prefabs/Gelato/GelatoModule.cs

```diff
@@ -0,0 +1,223 @@
+using System.Numerics;
+using System.Threading;
+using System.Threading.Tasks;
+using ChainSafe.GamingSDK.EVM.MetaMaskBrowserWallet;
+using ChainSafe.GamingWeb3.Build;
+using ChainSafe.GamingWeb3.Unity;
+using ChainSafe.GamingSdk.Gelato;
+using ChainSafe.GamingSdk.Gelato.Dto;
+using ChainSafe.GamingWeb3;
+using UnityEngine;
+using Web3Unity.Scripts.Library.Ethers.JsonRpc;
+using Contract = Web3Unity.Scripts.Library.Ethers.Contracts.Contract;
+
+public class GelatoModule : MonoBehaviour
```

Comment:
After #486 is merged in, update this script to use the `Web3Accessor` script. Also, web3 initialization should be moved to the login scene.

Reply:
@Arshia001 Where is the login screen in context of Gelato?